app/controllers/posts_controller.rb

```diff
@@ -509,7 +509,7 @@ def locked
     render_json_dump(locked: post.locked?)
   end
 
-  def notice
+  def notice_action
```

Comment:
See comment in `config/routes.rb`

Reply:
What do you think about `toggle_notice` instead? `notice_action` doesn't seem to properly describe what this controller method does.

Reply:
Glad to change its name, I wasn’t very happy with the current one :+1: